scoring-service-builder/.gradle/2.7/taskArtifacts/cache.properties

```diff
@@ -1 +0,0 @@
-#Tue Sep 27 09:56:58 PDT 2016
```

Comment:
Remove the scoring-service-builder files, the directory name was changed. You might be able to just pull from develop to pull in the change.